config/kernel/linux-meson64-edge.config

```diff
@@ -1381,7 +1402,6 @@ CONFIG_NF_TABLES_IPV6=y
 CONFIG_NFT_REJECT_IPV6=m
 CONFIG_NFT_DUP_IPV6=m
 CONFIG_NFT_FIB_IPV6=m
-CONFIG_NF_FLOW_TABLE_IPV6=m
```

Comment:
Why you disable 
CONFIG_NF_FLOW_TABLE_IPV6/IPV4 in kernel config?

Reply:
I can't find it in 5.17.1 kernel, so it disabled automatically.

Reply:
@adeepn NF_FLOW_TABLE_IPV4 and NF_FLOW_TABLE_IPV6 are invisble, selected by
nothing (so they can no longer be enabled), and their last real users
have been removed (nf_flow_table_ipv6.c is empty).
https://lkml.org/lkml/2022/1/23/86

Reply:
thanks.